docs/whats-new/release-notes/v3_3_0.md

```diff
@@ -10,10 +10,36 @@ See [New features and enhancements](#new-features-and-enhancements) for a full l
 
 Zowe Version 3.3.0 contains the enhancements that are described in the following topics.
 
-### Zowe installation and packaging
+### Server Install
+
+- A new utility `certificate-analyser.jar` will be added to the folder `zowe/bin/utils`. This tool will eventually be used to verify correct certificates for use by zowe, as well as correct connectivity to servers Zowe communicates with such as z/OSMF. In a future release it is intended to be used by Zowe under specific circumstances to verify whether the network setup is correct. ([#4354](https://github.com/zowe/zowe-install-packaging/pull/4354))
+- Enhanced the `ZWEGENER` job to support Configuration Manager path elements of type `PARMLIB` that can contain one or multiple members with different names within the same `PARMLIB` dataset. The new format is `PARMLIB(DATA.SET(member))`. This is an alternative format to the older format. ([#4332](https://github.com/zowe/zowe-install-packaging/pull/4332))
+- Updated the `zowe-yaml-schema.json` and `defaults.yaml` to support a new configuration parameter `zowe.sysMessageTrim` and set a default value. This parameter is used to trim syslog messages and print only from the sys-message-id. ([#4294](https://github.com/zowe/zowe-install-packaging/pull/4294))
+-	The `zwe init` commands now use `ZWEGENER` to create and submit JCL that modifies the system during Zowe’s installation and configuration. This replaces the previous method in which zwe init commands used USS commands and utilities. The `--dry-run` option was added and generates the JCL without submitting it, so users can review and optionally submit it manually.
+-	A new parameter `zowe.setup.jcl.header` can now be used to specify Job Card information in the JCL generated by `ZWEGENER`.
+- The zwe install command now uses Configuration Manager and JCL for the installation of Zowe server. The previous shell-script implementation has been removed. ([#4282](https://github.com/zowe/zowe-install-packaging/pull/4282))
```

Comment:
Same as comment above:

The `zwe install` command uses Configuration Manager and JCL for the installation of Zowe server as an alternative....

Reply:
This can optionally be merged with `zwe init`, as mentioned in [comment above](https://github.com/zowe/docs-site/pull/4686#discussion_r2322692501)